### PR TITLE
Changelog for v43.0.0

### PR DESCRIPTION
**Pull request by psmyrek**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal: Changelog for v43.0.0.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,257 @@
...
+
+There are exciting ways you can use merge fields:
+
+1. **Creating Templates:** Easily insert merge fields as placeholders. The default preview mode shows these placeholders, giving you a clear layout without any data distraction.
+2. **Previewing with Example Data:** Want to see how your document will look with actual data? Switch the preview mode to display sample data, giving you a sneak peek of the final result.
+3. **Editing Merged Documents:** Start with a blank document or a pre-made template, and see placeholders automatically fill with real data. This mode shows the actual data instead of placeholders, streamlining the editing process.
+
+Read more in the [merge fields documentation](https://ckeditor.com/docs/ckeditor5/latest/features/merge-fields.html).
+
```
Comment (scofalik) [inline CHANGELOG.md:21]:
```suggestion
The new [merge fields feature](https://ckeditor.com/docs/ckeditor5/latest/features/merge-fields.html) is a game-changer for creating document templates and other kinds of personalized or dynamic content. Thanks to this feature, you can insert placeholders into your content, indicating where actual values should go. These places are marked in the final content in a distinctive way, making it easy to later process the template and fill it with the actual values. The feature supports a preview mode too -- you can define preview data sets, and see how the content will look like when real values are used directly in the editor. The plugin is highly customizable to fit various applications and scenarios. Finally, merge fields are fully integrated with our Export to Word, Export to PDF, and Import from Word features, both when they are used from the editor and via REST API.

We are extremely happy to share with you this highly demanded feature, and we cannot wait to listen to your feedback!

Make sure to visit our [builder](https://ckeditor.com/ckeditor-5/builder/) or [documentation](https://ckeditor.com/docs/ckeditor5/latest/features/merge-fields.html) to learn more about the feature.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,257 @@
...
+#### Export to Word V2 as the default version
+
+The V2 version of the [export to Word feature](https://ckeditor.com/docs/ckeditor5/latest/features/converters/export-word.html) introduced significant improvements, optimizations, and fixes. This update enhances the overall performance, making the export process faster and more reliable, especially for large documents. Key improvements include better handling of table borders, automatic detection of Word styles from CSS, and support for more text-related CSS properties, ensuring your documents maintain their intended formatting.
+
+Starting this version, the V2 configuration becomes the default in the `ExportWord` plugin. Make sure to [migrate your configuration](https://ckeditor.com/docs/ckeditor5/latest/updating/guides/update-to-43.html#export-to-word-v2-becomes-the-default) if you are using it.
+
+#### Improved sidebar accessibility and navigation
+
+We have introduced multiple enhancements to make the sidebar more accessible. You can now use `Shift+Ctrl+E` to move focus from the editor to the active annotation. Navigation has been improved with `Arrow Up` and `Arrow Down` for annotations and comments archive.
```
Comment (scofalik) [inline CHANGELOG.md:30]:
```suggestion
We have introduced multiple enhancements to make the sidebar more accessible. Among other improvements, you can now use `Shift+Ctrl+E` to move focus from the editor marker to the active annotation. When your comment reply is ready, you can quickly submit it using a new `Ctrl+Enter` shortcut. Also, navigation from one annotation to another is now possible by using arrow keys.

The full lists of keyboard supported actions can be reviewed in our [Accessibility support guide](https://ckeditor.com/docs/ckeditor5/latest/features/accessibility.html#Keystrokes-for-interacting-with-annotation-threads-e-g-comments-track-changes-suggestions).
```

Comment (scofalik) [reply CHANGELOG.md:30]: I am not sure if the link is correct I can verify it later or we can simplify it.

**Review by scofalik — CHANGES_REQUESTED** [review]

**Comment by scofalik** [comment]
I will have a few more items for the changelog, please wait a bit.

**Review by scofalik — APPROVED** [review]

**Review by godai78 — APPROVED** [review]

**Pull request merged**